iina/QuickSettingViewController.swift

```diff
@@ -394,23 +404,19 @@ class QuickSettingViewController: NSViewController, NSTableViewDataSource, NSTab
     }
     let sliderValue = sender.doubleValue
     let value = AppData.minSpeed * pow((AppData.maxSpeed / AppData.minSpeed), sliderValue / sliderSteps)
-    let formattedValue = (customSpeedTextField.formatter as? NumberFormatter)?.string(from: NSNumber(value: value)) ?? ""
-    speedSliderIndicator.stringValue = "\(formattedValue)x"
-    customSpeedTextField.stringValue = formattedValue
+    customSpeedTextField.doubleValue = value
     playerCore.setSpeed(value)
-    redraw(indicator: speedSliderIndicator, constraint: speedSliderConstraint, slider: speedSlider, value: "\(formattedValue)x")
+    redraw(indicator: speedSliderIndicator, constraint: speedSliderConstraint, slider: speedSlider, value: "\(customSpeedTextField.stringValue)x")
   }
 
   @IBAction func customSpeedEditFinishedAction(_ sender: NSTextField) {
```

Comment:
The other text fields default to "0" when nothing is entered; I believe that we should make a similar argument for this text field but have it default to "1" (normal speed) instead.